scripts/gen_dockerfile.py

```diff
@@ -70,7 +62,7 @@
 # Validated application configuration
 AppConfig = collections.namedtuple(
     'AppConfig',
-    'base_image dockerfile_python_version entrypoint has_requirements_txt'
+    'base_image dockerfile_python_version entrypoint has_requirements_txt, is_python_compat'
```

Comment:
Can you remove the comma for consistency (or add commas to all the others)?  Apparently any run of whitespace and commas are acceptable to namedtuple.